ddtrace/constants.py

```diff
@@ -38,32 +37,4 @@
 # Use this to explicitly inform the backend that a trace should be kept and stored.
 USER_KEEP = 2
 
-
-_DEPRECATED = {
-    # TODO: Removal of this attribute does not require the addition of a
-    # constant as it has only a single use for passing trace filters in the
-    # settings to the tracer.
-    "FILTERS_KEY": "FILTERS",
-    # TODO: Moved to other modules but cannot reference here due to circular imports
-    # ddtrace.contrib.logging.patch._LOG_SPAN_KEY
-    # ddtrace.span._NUMERIC_TAGS
-    "NUMERIC_TAGS": (ANALYTICS_SAMPLE_RATE_KEY,),
-    "LOG_SPAN_KEY": "__datadog_log_span",
-}
-
-
-def __getattr__(name):
-    if name in _DEPRECATED:
-        debtcollector.deprecate(
-            ("%s.%s is deprecated" % (__name__, name)),
-            removal_version="1.0.0",
-        )
-        return _DEPRECATED[name]
-
-    if name in globals():
-        return globals()[name]
-
-    raise AttributeError("%s has no attribute %s", __name__, name)
-
-
 ensure_pep562(__name__)
```

Comment:
This needs to go away too. This just makes sure `__getattr__` works as intended

Reply:
Good catch. I was able to remove the pep562 dependency :)